docker/README.md

```diff
@@ -19,7 +19,7 @@ Static binaries building using custom alpine image.
 image defined in [Dockerfile](build/Dockerfile). In order to build them you should specify
 `OCTEZ_VERSION` env variable and run the script:
 ```
-export OCTEZ_VERSION="v14.1"
+export OCTEZ_VERSION="v17.3"
```

Comment:
These are meant to be example commands and we don't update them for every Octez update, so I think they are fine either way.

TBH I find this commit unnecessary, but I don't mind it either, so feel free to keep it